GT-127 Outdated links to be configurable

rowan04 opened the pull request:
Resolves #433

Adds configurable options for projects, ngis and service groups.

rowan04 committed:
Adds configurable options for documentation links for Projects, NGIs and Service Groups

- changes to local_info.xml and Config.php to allow configurable links

rowan04 committed:
Changes to project, ngi and service group files to allow for configurable links

- changes to controllers to add configurable link to params array
- changes to views to use the params array

rowan04 commented:
This is similar too and will conflict with #412

gregcorbett reviewed (dismissed):
LGTM

gregcorbett commented:
I want to make the necassary config change, in a sandbox, at the same time before merging this.

gregcorbett commented:
rebasing to update

gregcorbett committed:
Update local_info schema with the new elements

- new configurable elements have been added to local_info.xml to
  allow documentation links to be configurable (and not
  hardcoded).
- These new elements need corresponding schema definitions to
  ensure an updated local_info.xml continues to pass the
  validation check.

gregcorbett committed:
Shorten long lines as detected by codacy

tofu-rocketry reviewed (commented):
Could you clarify the purpose of [357fa82](https://github.com/GOCDB/gocdb/pull/436/commits/357fa8260a36286aa29ead6aac006f044a0e380c)?

gregcorbett commented:
> Could you clarify the purpose of [357fa82](https://github.com/GOCDB/gocdb/pull/436/commits/357fa8260a36286aa29ead6aac006f044a0e380c)?

I've updated the commit message (now commit https://github.com/GOCDB/gocdb/pull/436/commits/4a9544f74c2dcaf090f7496a9f0400491dfa0e2b)

tofu-rocketry commented:
Thanks. Your commits look ok now. I can green tick this if you like, but on the understanding I've not got to grips with the rest of the PR.

gregcorbett commented:
Sound good to me 👍

tofu-rocketry reviewed (approved):
Limited approval for @gregcorbett 's additional commits.
